internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -28,20 +29,17 @@ const (
 )
 
 const (
-	annotationCPULoadBalancing = "cpu-load-balancing.crio.io"
-	annotationCPUQuota         = "cpu-quota.crio.io"
-	annotationIRQLoadBalancing = "irq-load-balancing.crio.io"
-	annotationTrue             = "true"
-	annotationDisable          = "disable"
-	schedDomainDir             = "/proc/sys/kernel/sched_domain"
-	irqSmpAffinityProcFile     = "/proc/irq/default_smp_affinity"
-	cgroupMountPoint           = "/sys/fs/cgroup"
+	annotationTrue         = "true"
+	annotationDisable      = "disable"
+	schedDomainDir         = "/proc/sys/kernel/sched_domain"
+	irqSmpAffinityProcFile = "/proc/irq/default_smp_affinity"
+	cgroupMountPoint       = "/sys/fs/cgroup"
 )
 
 // HighPerformanceHooks used to run additional hooks that will configure a system for the latency sensitive workloads
 type HighPerformanceHooks struct{}
 
-func (h *HighPerformanceHooks) PreStart(ctx context.Context, c *oci.Container, s *sandbox.Sandbox) error {
+func (h *HighPerformanceHooks) PreStart(ctx context.Context, c *oci.Container, s *sandbox.Sandbox, r *oci.Runtime) error {
```

Comment:
AFAICT, we still only add the HighPerformanceHooks object to the container hooks on a string match wtih the runtime. We'd have to go through a slightly different design and unconditionally call these hooks (and they'd branch here whether the hooks are actually used)

Reply:
a metric of success of this PR is if we can add any of these annotations in any old runtime handler (default, for instance) and they'd be interpreted, regardless of the name of the runtime handler